spec/rubocop/cop/style/raise_args_spec.rb

```diff
@@ -322,6 +322,22 @@ def do_something
       expect_no_offenses('raise MyCustomError.new(*args)')
     end
 
+    context 'when Ruby >= 3.2', :ruby32 do
+      it 'accepts a raise with anon splatted arguments' do
```

Comment:
That's just my two cents:

```suggestion
      it 'accepts a raise with anonymous splatted arguments' do
```